test/flows/find_coronavirus_support_flow_test.rb

```diff
@@ -0,0 +1,597 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class FindCoronavirusSupportFlowFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow FindCoronavirusSupportFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: need_help_with" do
+    setup { testing_node :need_help_with }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of nation for an empty response" do
+        assert_next_node :nation, for_response: %w[none]
+      end
+
+      should "have a next node based on the response" do
+        assert_next_node :self_employed, for_response: %w[being_unemployed mental_health]
+      end
+    end
+  end
+
+  context "question: feel_unsafe" do
+    setup do
+      testing_node :feel_unsafe
+      add_responses need_help_with: %w[feeling_unsafe]
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    %w[yes concerned_about_others no not_sure].each do |response|
+      should "have a next node of nation if response is #{response} and there aren't any other needs_help_with questions" do
+        assert_next_node :nation, for_response: response
+      end
+
+      should "have next node of next question group for #{response} if there are other needs_help_with questions" do
+        add_responses need_help_with: %w[feeling_unsafe mental_health]
+
+        assert_next_node :mental_health_worries, for_response: response
+      end
+    end
+  end
+
+  context "question: afford_rent_mortgage_bills" do
+    setup do
+      testing_node :afford_rent_mortgage_bills
+      add_responses need_help_with: %w[paying_bills]
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    %w[yes no not_sure].each do |response|
+      should "have a next node of nation if response is #{response} and there aren't any other needs_help_with questions" do
+        assert_next_node :nation, for_response: response
+      end
+
+      should "have next node of next question group for #{response} if there are other needs_help_with questions" do
```

Comment:
I think we're okay to change "next question group" to "mental health worries" given that it's hardcoded below. Figure it makes the test titles a bit clearer. Applies to all the tests so I'm not going to repeat this over and over

Reply:
ah, ignore this comment, I've realised why i'm wrong 🤦